pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -87,12 +94,17 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 	if len(m.nodeConditions) == 0 {
 		return lifecycle.PodAdmitResult{Admit: true}
 	}
-	notBestEffort := qos.BestEffort != qos.GetPodQOS(attrs.Pod)
-	if notBestEffort {
-		return lifecycle.PodAdmitResult{Admit: true}
+
```

Comment:
Where do you plan to invoke image GC when there's disk pressure?(https://github.com/kubernetes/kubernetes/blob/186c0c7b14dfc9403b131b68b70da08227c71a04/pkg/kubelet/image_manager.go#L51 provides the method to delete all unused images.)